src/lib/utils/preconditions/PreconditionContainerArray.ts

```diff
@@ -13,7 +13,7 @@ import { PreconditionContainerSingle, PreconditionSingleResolvable } from './Pre
  */
 export const enum PreconditionRunMode {
 	/**
-	 * The entries are run sequentially, this is the default behaviour and can be slow when doing long asynchronous
+	 * The entries are run sequentially, this is the default behavior and can be slow when doing long asynchronous
```

Comment:
```suggestion
	 * The entries are run sequentially, this is the default behaviour and can be slow when doing long asynchronous
```

Oi you bloody wanka, don't cha speak the Queen's English mate?

Reply:
Ew